users/templates/users/account_settings.html

```diff
@@ -1,165 +1,159 @@
+                                {#                        <h4 class="fw-bolder fs-6">Connect to Twitter</h4>#}
+                                {#                        <p class="text-light-emphasis">Connect a Twitter account to enable the choice to tweet your new posts and display a link on your profile. We will never post to Twitter without your permission.</p>#}
+                                {#                    </div>#}
+                                {#                    <div class="col d-flex justify-content-end">#}
+                                {#                        <button type="button" class="btn btn-dark border-light rounded-pill">Connect to Twitter</button>#}
+                                {#                        <!-- Add logic for adding Twitter account -->#}
+                                {#                    </div>#}
+                                {#                </div>#}
+                                {#                <div class="row align-items-center">#}
+                                {#                    <div class="col d-flex flex-column flex-fill">#}
+                                {#                        <h4 class="fw-bolder fs-6 text-light-emphasis">Show link on profile</h4>#}
+                                {#                    </div>#}
+                                {#                    <div class="col d-flex justify-content-end">#}
+                                {#                        <div class="form-check form-switch">#}
+                                {#                            <input class="form-check-input" type="checkbox" role="switch" id="showTwitterProfile" />#}
+                                {#                            <label class="form-check-label text-light-emphasis" for="showTwitterProfile">You can show a link to your Twitter account on your profile</label>#}
+                                {#                        </div>#}
+                                {#                    </div>#}
+                                {#                </div>#}
+                                {#            </div>#}
+
+                                <!-- Opt into beta tests from form -->
                                 <h3 class="mt-4 mb-4 text-secondary fs-6 border-bottom border-secondary">Beta tests</h3>
                                 <h4 class="fw-bolder fs-6 mb-3">Opt into beta tests</h4>
                                 <div class="form-check form-switch mb-3">
-                                    <input class="form-check-input" type="checkbox" role="switch" id="optInBetaTests" />
-                                    <label class="form-check-label text-light-emphasis" for="optInBetaTests">See the newest features from Reddit and join the r/beta community</label>
+                                    <input type="checkbox" class="form-check-input" name="is_beta"
+                                           {% if  form.is_beta.value %}checked{% endif %}>
```

Comment:
Hint, not required: check if there is crispy forms handler for checkboxes. Might simplify it.

Reply:
```
{{ form.is_beta|as_crispy_field }}
```

Reply:
Done